docs/examples/basic.md

```diff
@@ -0,0 +1,88 @@
+# Basic OTG Configurations
+
+## Single TCP Traffic Flow between two Test Ports
+
+File: [`flow.otg.yaml`](flow.otg.yaml).
+
+This single flow configuration specifies:
+
+* Flow Name: `f1`
+* Packet Schedule: 
+    * `1000` packets 
+    * of `512B` size 
+    * to be transmitted as fixed rate of `100pps`
+* Flow Template: 
+    * Ethernet src `02:00:00:00:01:aa` dst `02:00:00:00:02:aa`
+    * IPv4 src `192.0.2.1` dst `192.0.2.2`
+    * TCP ports src `1024 x 7 x increment` dst `7`
+* Test Ports
+    * Tx: `p1` with OTG location `localhost:5555`
+    * Rx: `p2` with OTG location `localhost:5556`
+
+This configuration was generated by running:
+
+```Shell
+otgen create flow --rate 100 --size 512 > flow.otg.yaml
+```
+
+## Two Traffic Flows representing bi-directional UDP traffic between two Traffic Ports
+
+File: [`bidir-flow.otg.yaml`](bidir-flow.otg.yaml).
+
+This bi-directional flow configuration specifies:
+
+* Flow Names: `f1`; `f2`
+* Packet Schedule: 
+    * `f1: 1000`; `f2: 100` packets
+    * of `f1: 512B`; `f2: 256B` size 
+    * to be transmitted as fixed rate of `f1: 100pps`; `f2: 10pps`
+* Flow Template: 
```

Comment:
If you change the previous occurrence to `Packet Template`, change this one too.